mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -324,6 +365,53 @@ func generateBigtableGCPolicy(d *schema.ResourceData) (bigtable.GCPolicy, error)
 	return policies[0], nil
 }
 
+func getGCPolicyFromJSON(p map[string]interface{}) (bigtable.GCPolicy, error) {
+	policies := []bigtable.GCPolicy{}
+	if p["rules"] == nil {
+		return bigtable.NoGcPolicy(), nil
+	}
+
+	for _, raw := range p["rules"].([]interface{}) {
```

Comment:
nit: for _, rules := range p["rules"].([]interface{}) {

Reply:
Done.